### ISSUE TITLE
permissionsView: unexpected forbidden

### ISSUE THREAD

**Issue by smadbe** [France-ioi/AlgoreaBackend#800]
This request 
```
curl 'https://dev.algorea.org/api/groups/672913018859223173/permissions/670968966872011405/314613032161178344' -H 'Authorization: Bearer ***' 
```
returns a `403 forbidden` while:
* the source group `672913018859223173`: `... current_user_can_grant_group_access: true `
* group: `670968966872011405` is the user himself and descendant of `672913018859223173` 
* item `314613032161178344` `can_grant_view: "content_with_descendants"`

**Comment by zenovich** [comment]
Cannot reproduce. Probably, it has been fixed by #803

**Comment by smadbe** [comment]
Not returning 403 anymore.
